src/main/java/com/aquaticinformatics/aquarius/sdk/timeseries/servicemodels/Acquisition.java

```diff
@@ -823,6 +823,6 @@ public static class PostVisitFileBase implements IFileUploadRequest
 
     public static class Current
     {
-        public static final AquariusServerVersion Version = AquariusServerVersion.Create("22.3.75.0");
+        public static final AquariusServerVersion Version = AquariusServerVersion.Create("23.1.61.0");
```

Comment:
This doesn't need to be the 23.1 GA version right? Just checking :)

Reply:
No. It needs to be whatever version past which there were no API changes until the GA release.
Though if we need to make some changes say, addressing a bug in regression sprint that changes the API signature. We'd need to do this process again and publish new version of SDK, not that big deal.